backend/blueprint/api/models/user.py

```diff
@@ -6,8 +6,8 @@
 class User(db.Model):
     __tablename__ = 'User'
 
-    id = db.Column(db.String(100), primary_key=True)
-    username = db.Column(db.String(80))
+    id = db.Column(db.String(100))
+    username = db.Column(db.String(80), primary_key=True)
```

Comment:
one can add `unique=True`

Reply:
both are different.
`primary_key` will be used here.
cheers!